webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -149,6 +506,47 @@
       .toUpperCase();
   }
 
+  onEditorInit(event: any): void {
+    // Access the TinyMCE editor instance
+    this.editorInstance = event.editor as Quill;
+
+    this.editorInstance.clipboard.dangerouslyPasteHTML(
+      this.selectedCourse.description
+    );
+    const toolbarModule: any = this.editorInstance.getModule('toolbar');
+    if (toolbarModule && toolbarModule.container && toolbarModule.container.querySelector('.ql-image')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)